src/common/components/Button/index.styl

```diff
@@ -0,0 +1,12 @@
+@require 'cozy-ui'
+
+@media (max-width: (1023/basefont)rem)
```

Comment:
Might as well use the new syntax for MQs.
`+medium-screen()` instead of `@media (max-width: (1023/basefont)rem)`

Reply:
Didn't knew these mixins exist. But https://github.com/cozy/cozy-drive/pull/449#discussion_r147335530 ?